core/arch/arm/sm/sm_a32.S

```diff
@@ -406,7 +406,7 @@ FUNC sm_init , :
 
 	bx	lr
 END_FUNC sm_init
-DECLARE_KEEP_PAGER sm_init
+DECLARE_KEEP_INIT sm_init
```

Comment:
We can probably remove `DECLARE_KEEP_INIT()` too. It will however still have to remain in the unpaged area since secondary CPUs may initialize after the primary CPU has started paging.

Reply:
right!